Updated contribute.adoc with Salesforce CLA (#105)
mulesoft/docs-general: modules/ROOT/pages/contribute.adoc

To contribute to the English version of this documentation:

> . Sign the https://cla.salesforce.com/sign-cla[Salesforce.com Contributor License Agreement (CLA)] if this is your first time contributing. The CLA only needs to be signed once per GitHub account.

Reviewer: I think the sentence structure might be better if you switch the order of the clauses; "If this is your first time ..., sign the ..."

Author: I agree, feels weird.

> . If this is your first time contributing, sign the https://cla.salesforce.com/sign-cla[Salesforce.com Contributor License Agreement (CLA)]. The CLA only needs to be signed once per GitHub account.

Reviewer 2: Placement of "only"; see associated entry in https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+Writing+Style+Reference.

```suggestion
. If this is your first time contributing, sign the https://cla.salesforce.com/sign-cla[Salesforce.com Contributor License Agreement (CLA)]. The CLA needs to be signed only once per GitHub account.
```

> . Click *Edit in Github* on the page you want to modify.

Reviewer 2: Because I am guessing that there is user navigation involved between the first step and this, I suggest locating the user first:
```suggestion
. On the page you want to modify, click *Edit in Github*.
```
OR
```suggestion
. Navigate to the page that you want to modifiy and click *Edit in Github*.
```

Reviewer 2: By the way, we have not talked about this yet, but you own your work. Period.

If I make a grammar or style or copy edit, you should definitely implement those (if you think you have an exception, and there are some, please ask). 

The rewordings that I suggest are for you to evaluate based on your own knowledge of your intent and technical precision. 

The questions that I ask are not for you to answer for me, but for your reader: please similarly consider them from the audience point of view and adjust text as you see fit.

We can discuss this further if you like!

. Make your edits.
. Submit a pull request to the Git repository.

> If you submit the pull request without signing the CLA first, you will receive an error in the pull request. Sign the CLA to remove this error.

Reviewer 2: This seems gratuitous to me, given Step 1 of the task.
If we must include it because Support has found that this is a common occurrence, then maybe more of a reminder, something like the following?
```suggestion
If your pull request results in an error, verify that you completed Step 1.
```
Are other errors likely? If so, we might want to add something like "...Step 1 before asking for help."

> The Documentation team will either accept your pull request or reach out if there are questions.

Reviewer 2: We try to avoid verb tenses other than present.
```suggestion
The Documentation team either accepts your pull request or contacts you for clarification.
```
We also try to avoid idioms, like "reach out," just to make things a bit easier for translation.
One more thing to avoid is "there is" and "there are" when they act as pseudo-subjects; see Style Ref.
